docs/developer/architecture/service-observability.md

```
@@ -114,3 +114,115 @@ func (t *TracedService) ListCertificateSigningRequests(ctx context.Context, p ap
+- `SIGTERM` - Graceful termination (default for `kubectl delete pod`)
+- `SIGQUIT` - Quit with core dump
+
+**Signal Processing:**
+- Uses `signal.NotifyContext()` for cross-platform compatibility
+- 30-second timeout for graceful shutdown operations
+- Tracer shutdown happens automatically via defer before timeout
+
+### Shutdown Observability
+
+**Structured Logging:**
+```go
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)